draft-ietf-babel-information-model.md

```diff
@@ -648,42 +789,178 @@ babel-route-selected:
   is being advertised).
 
 
-# Common Objects
+## Definition of babel-hmac-obj
+
+~~~~
+  object {
+       string                rw babel-hmac-algorithm;
+       boolean               rw babel-hmac-verify-received;
+       string                rw babel-hmac-interfaces<0..*>;
+       babel-hmac-keys-obj   rw babel-hmac-keys<0..*>;
+   } babel-hmac-obj;
+~~~~
+{: artwork-align="left"}
+
+
+babel-hmac-algorithm
+: The name of the HMAC algorithm this object
+  instance uses. The value MUST be the same as one of the enumerations
+  listed in the babel-hmac-algorithms parameter.
+  An implementation MAY choose
+  to expose this parameter as read-only ("ro").

```

Comment:
Again I find a conflict between this being a MUST and it being a ro parameter.

Reply:
Same response about report-only implementations. Not all implementations should be assumed to be used with fully functional management protocols.